src/report-table/report-table.component.tsx

```diff
@@ -143,13 +156,35 @@ const ReportTable = (props: ReportTableProps) => {
                                     <Link
                                       href={getReportUrl(rows, row.id)}
                                       target={'_blank'}
+                                      onClick={() => {
+                                        const auditMessage = getAuditMessageBody(
+                                          patientUuid,
+                                          row.cells[2].value,
+                                          row.cells[0].value,
```

Comment:
what are these magical index references?

Reply:
since report table was using pagination, this was used to get the file name, test name and report date. It has been changed so that now the position of row in table is calculated and that is being used to get necessary fields.